### DISCUSSION TITLE
Concourse's public metrics dashboard and metrics emitted

### DISCUSSION THREAD

**Discussion by dlbock** [concourse/concourse#7699] [General]
Hi folks, I have 2 questions regarding [Concourse metrics](https://concourse-ci.org/metrics.html)

- Is the list of metrics mentioned in ☝️ that page up-to-date? I see some metrics showing up on our `web` pods that aren't in that list (e.g. `concourse_builds_duration_seconds_bucket`, `concourse_builds_duration_seconds_sum`, `concourse_builds_duration_seconds_count`), and some metrics mentioned on that ☝️ page not showing up for us (e.g. `concourse_tasks_waiting`, `concourse_tasks_wait_duration`)
- it seems the [metrics dashboard](https://p.datadoghq.com/sb/bb663c3fb-c0f3ba68561631a1b130a4507bf36a15?tpl_var_environment=ci&tpl_var_web=ci-web&tpl_var_worker=ci-worker&from_ts=1634588342976&to_ts=1634591942976&live=true) linked from the aforementioned page does not seem to contain any data anymore?

Maybe I'm missing something? 🙏

**Comment by taylorsilva** [comment]
That page is fairly up to date. One thing I noticed the last time I updated it, which is probably over a year ago now 😅 , is that the promethus page we scrape from doesn't list every metric. There are some metrics that it only displays once the metric has a value to show. Not all metrics were like this, only some. I couldn't figure out why this happened but it's probably why you're finding extra metrics in your own deployment.
